### PR TITLE
WIP: Update dockerfile with arguments

### PR DESCRIPTION
**Pull request by CODeRUS**

### CODE REVIEW

**Comment by CODeRUS** [comment]
TBD

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
+
+COPY sdk_bash_completion.sh /etc/profile.d/
+
+USER user
+
+RUN sudo zypper ref
+RUN sudo zypper -qn in $extra_packages
+
+RUN sdk-assistant -y create $name_tooling $tooling_url
 COPY mer-tooling-chroot /srv/mer/toolings/SailfishOS-latest/
```
Comment (SfietKonstantin) [inline Dockerfile]: This should be /srv/mer/toolings/${name_tooling}

File: Dockerfile [hunk 2]
```diff
@@ -1,18 +1,34 @@
 FROM sailfishos-platform-sdk-base
 MAINTAINER Lucien Xu <sfietkonstantin@free.fr>
 
-# Add nemo in sudoers without password
-RUN chmod +w /etc/sudoers && echo "nemo ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers && chmod -w /etc/sudoers
+ARG extra_packages
```
Comment (SfietKonstantin) [inline Dockerfile:4]: I'm not sure about the need of this. Tar is needed to untar the sdk images, so more like a build dependency requirement.

File: build.sh [hunk 3]
```diff
@@ -3,20 +3,49 @@ set -euo pipefail
...
+bzip2 -d "$SDK_NAME.tar.bz2"
 echo "Extracting base image: DONE"
 
+fi
+
+
 # Import it as a docker base image & build the full image
 echo "Importing base image"
-docker import "${SDK_NAME}.tar" sailfishos-platform-sdk-base
+docker import "$SDK_NAME.tar" sailfishos-platform-sdk-base
```
Comment (SfietKonstantin) [inline build.sh:36]: You don't like braces :p ?

**Review by SfietKonstantin — CHANGES_REQUESTED** [review]

**Comment by SfietKonstantin** [comment]
Oups, I might have reviewed too early, sorry ...

File: Dockerfile (hunk 2 above)
Comment (CODeRUS) [reply Dockerfile:4]: As option one can include openssh-server here

File: build.sh (hunk 3 above)
Comment (CODeRUS) [reply build.sh:36]: Nah, I'm using bash best practices. Too many brackets.

**Comment by CODeRUS** [comment]
Also I'm thinking about additional steps for adding/modifying repositories lists inside targets.

File: Dockerfile (hunk 1 above)
Comment (CODeRUS) [reply Dockerfile]: done

**Pull request closed without merging**